Documentation/Blazorise.Docs/Pages/Blog/2024-03-26_HowToOverrideFluentDesignTokens/Index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to override Fluent design tokens
+description: In this blog post, we'll guide you through the steps to properly override Fluent design tokens for use in your Blazorise project, even with the current limitations in Blazorise v1.5.
+permalink: /blog/how-to-override-fluent-design-tokens
+canonical: /blog/how-to-override-fluent-design-tokens
+image-url: /img/blog/2024-03-26/how-to-override-fluent-design-tokens.png
+image-title: How to override Fluent design tokens
+author-name: Mladen Macanovic
+author-image: mladen
+posted-on: March 26th, 2024
+read-time: 5 min
+---
+
+# How to override Fluent design tokens
+
+Creating a visually appealing and user-friendly interface is crucial for any web application. [Microsoft's Fluent Design System](https://fluent2.microsoft.design/) provides a robust framework for creating such interfaces, offering a wide array of design tokens that can be utilized to craft beautiful and intuitive UIs.
+
+However, when it comes to integrating these design tokens into a Blazorise project, the process is not as straightforward as one might hope. In this blog post, we'll guide you through the steps to properly override Fluent design tokens for use in your Blazorise project, even with the current limitations in Blazorise v1.5.
```

Comment:
I personally still don't like 
"even with the current limitations in Blazorise v1.5." 

If someone he's already reading this, he'll get to it in the next ## The Problem section?